msg-transport/src/ipc/mod.rs

```diff
@@ -92,8 +92,18 @@ impl PeerAddress<PathBuf> for IpcStream {
     }
 }
 
+impl TryFrom<&IpcStream> for () {
+    type Error = std::io::Error;
+
+    fn try_from(_: &IpcStream) -> Result<Self, Self::Error> {
+        Ok(())
+    }
+}
```

Comment:
Can you add a similar TODO comment here? Reading this feels I'm missing some context.